Signal/src/view controllers/PrivacySettingsTableViewController.m

```diff
@@ -123,6 +156,11 @@ - (nullable NSString *)tableView:(UITableView *)tableView titleForFooterInSectio
         case PrivacySettingsTableViewControllerSectionIndexCalling:
             return NSLocalizedString(@"SETTINGS_CALLING_HIDES_IP_ADDRESS_PREFERENCE_TITLE_DETAIL",
                 @"User settings section footer, a detailed explanation");
+        case PrivacySettingsTableViewControllerSectionIndexCallKitEnabled:
```

Comment:
We need to hide these footers conditionally upon CallKit being disabled or not supported. e.g. I see both footers (but neither cell) on my iOS9 device.